tests/test_sync.py

```diff
@@ -25,7 +29,11 @@ def test_trie_sync(random):
     while len(requests) > 0:
         results = []
         for request in requests:
-            results.append([request.node_key, src_trie.db[request.node_key]])
+            if len(request.node_key) == 32:
+                results.append([request.node_key, src_trie.db[request.node_key]])
+            else:
+                import rlp
```

Comment:
Why import rlp here and not as a global?

Reply:
accidentally left in.  code path no longer gets traversed, removing.